### PR TITLE
ROADMAP: Follow ups on recent review

### PR DESCRIPTION
**Pull request by mxinden**
# Description

Follow ups to @BigLep's review in https://github.com/libp2p/rust-libp2p/pull/2997#pullrequestreview-1146692284.

- bf4d1ae35 ROADMAP: Prioritize QUIC beyond the first iteration
- 3aa4224a8 ROADMAP: Link to specs/ROADMAP.md
- 9d4728408 ROADMAP: Remove TLS
- 538619310 ROADMAP: Encourage feedback
- b8cbf2e47 ROADMAP: Remove project board for now

What do folks think?

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -25,16 +25,6 @@ This is a living document. Input is always welcome e.g. via GitHub issues or pul
...
 
-## TLS support
-
-| Category     | Status      | Target Completion | Tracking                                        | Dependencies                       | Dependents |
-|--------------|-------------|-------------------|-------------------------------------------------|------------------------------------|------------|
-| Connectivity | In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/pull/2945 | [QUIC](#experimental-quic-support) |            |
-
-
-This allows us to secure both TCP and QUIC connections using TLS. This is a requirement for QUIC
-support. Running TLS on top of TCP is a nice to have, since we already have noise.
```
Comment (thomaseizinger) [inline ROADMAP.md:36]: Should we mark it as completed instead? Or do you see the ROADMAP as something that only contains the remaining work?

**Review by thomaseizinger — APPROVED** [review]

File: ROADMAP.md [hunk 2]
```diff
@@ -25,16 +26,6 @@ For a more detailed view see https://github.com/orgs/libp2p/projects/4/views/1.
...
-
 ## Experimental QUIC support
```
Comment (elenaf9) [inline ROADMAP.md]: Should we still label it as _Experimental_ if the item goes beyond the first implementation?

Comment (mxinden) [reply ROADMAP.md]: :+1: 90e503fb8

File: ROADMAP.md (hunk 1 above)
Comment (mxinden) [reply ROADMAP.md:36]: > Or do you see the ROADMAP as something that only contains the remaining work?

I prefer that, yes.

Ideally, our release notes give a good timeline of things that got done. (I know they don't quite serve that purpose yet.)

**Comment by mxinden** [comment]
Merging here for a Protocol Labs roadmap session. Feel free to post more comments. I will make sure to include them in future pull requests.

**Review by jxs — APPROVED** [review]

File: ROADMAP.md [hunk 3]
```diff
@@ -25,29 +26,16 @@ For a more detailed view see https://github.com/orgs/libp2p/projects/4/views/1.
...
-
-This allows us to secure both TCP and QUIC connections using TLS. This is a requirement for QUIC
-support. Running TLS on top of TCP is a nice to have, since we already have noise.
-
-## Experimental QUIC support
+## QUIC support
 
 | Category     | Status      | Target Completion | Tracking                                          | Dependencies                                                        | Dependents |
 |--------------|-------------|-------------------|---------------------------------------------------|---------------------------------------------------------------------|------------|
 | Connectivity | In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/issues/2883 | [TLS](#tls-support), https://github.com/libp2p/test-plans/issues/53 |            |
```
Comment (BigLep) [inline ROADMAP.md:33]: I can't add a suggestion since this has been committed, but "[TLS](#tls-support)" is no longer a valid section.

**Review by BigLep — COMMENTED** [review]
FYI that the roadmap doesn't render as markdown: https://github.com/libp2p/rust-libp2p/blob/master/ROADMAP.md

File: ROADMAP.md (hunk 3 above)
Comment (BigLep) [reply ROADMAP.md:33]: Addressed in https://github.com/libp2p/rust-libp2p/pull/3091

**Comment by BigLep** [comment]
I put some followups in https://github.com/libp2p/rust-libp2p/pull/3091

**Pull request merged**